content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -94,98 +119,170 @@ There may be some delays due to caching and other propagation overhead.
 Remove the following configuration before you proceed to the next task:
 
 {{< text bash >}}
-$ kubectl delete -f @samples/bookinfo/platform/kube/rbac/namespace-policy.yaml@
+$ kubectl delete authorizationpolicy.security.istio.io/bookinfo-viewer
 {{< /text >}}
 
-## Enforcing Service-level access control
+## Enforcing workload-level access control
 
-This task shows you how to set up service-level access control using Istio authorization. Before you start, please make sure that:
+This task shows you how to set up workload-level access control using Istio authorization.
 
-* You have [enabled Istio authorization](#enabling-istio-authorization).
-* You have [removed namespace-level authorization policy](#cleanup-namespace-level-access-control).
+### applying a default `deny-all` policy
 
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see `"RBAC: access denied"`.
-We will incrementally add access permission to the services in the Bookinfo sample.
+In this step, we will create a default policy that denies all requests to workload in the default namespace.
 
-### Step 1. allowing access to the `productpage` service
+Run the following command:
 
-In this step, we will create a policy that allows external requests to access the `productpage` service via Ingress.
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+spec:
+  {}
+EOF
+{{< /text >}}
 
-Run the following command:
+The command creates a `deny-all` policy with no workload selector, the policy will select all workloads in the default namespace.
+
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see `"RBAC: access denied"`.
+This is because Istio authorization is "deny by default" and the `deny-all` policy doesn't have any rules to allow any access to
+workloads in the default namespace.
 
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step1_apply.sh" >}}
+{{< tip >}}
```

Comment:
This tip should be served at the beginning of the task since readers need to be aware that after applying each configuration they need to wait for it to propagate.

Reply:
Moved the tip to the beginning of the task.